Quotient/room.cpp

```diff
@@ -1777,6 +1777,7 @@ void Room::Private::updateThread(const RoomEvent* event)
     }
 
     auto& thread = threads[rme->threadRootEventId()];
+    const auto isNew = thread.threadRpotId.isEmpty();
```

Comment:
CI fails because of the typo.
```suggestion
    const auto isNew = thread.threadRootId.isEmpty();
```